TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -106,7 +68,10 @@ function Execute-SqlQueryDeployment
             }
         }
 
-        Invoke-Sqlcmd @spaltArguments
+        $additionalArguments = $additionalArguments.Replace("`$", "```$")
```

Comment:
Before it escaping was handled for other characters like quotes also and before doing remote.